<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuestWindow</class>
 <widget class="QMainWindow" name="GuestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>880</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tableTab">
     <attribute name="title">
      <string>Select Table</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="dishTab">
     <attribute name="title">
      <string>Select Dishes</string>
     </attribute>
     <widget class="QScrollArea" name="scrollAreaDishList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>850</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>850</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="ItemList" name="dishList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1198</width>
         <height>848</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="cartTab">
     <attribute name="title">
      <string>View Cart</string>
     </attribute>
     <widget class="QScrollArea" name="scrollAreaCartList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>1200</width>
        <height>720</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>720</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="ItemList" name="cartList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1198</width>
         <height>715</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="RefreshCart">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>790</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>View Cart</string>
      </property>
     </widget>
     <widget class="QLabel" name="textTotal">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>780</y>
        <width>61</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitButton">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>790</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="cartTotal">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>780</y>
        <width>201</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ff0000</string>
      </property>
      <property name="text">
       <string>ï¿¥0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="checkOutButton">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>790</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Check Out</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="msgTab">
     <attribute name="title">
      <string>Message Center</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="selfTab">
     <attribute name="title">
      <string>About Me</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ItemList</class>
   <extends>QWidget</extends>
   <header>itemlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
